lessons/Data-Science/filtering-and-building/langs/en-us/index.adoc

```diff
@@ -127,18 +127,15 @@ Let's think about what filtering and building would look like with Animals at th
 - Match the code on @printable-exercise{what-table-do-we-get-matching.adoc} with the descriptions of the tables they would produce.
 }
 
-@teacher{@opt{Give students some scaffolded practice filtering and building with the Animals! +
+@teacher{
+@opt{Give students some scaffolded practice filtering and building with the Animals! +
 
-- Practice using the Design Recipe to write new helper functions:   
-  * @opt-printable-exercise{design-recipe-1.adoc}
-  * @opt-printable-exercise{design-recipe-2.adoc}.
 - Practice using Circles of Evaluation with Filtering and Building to make:
   * @opt-printable-exercise{coe-table-old-cats.adoc, a Table of Old Cats} 
   * @opt-printable-exercise{coe-box-plot-young-dog-kilos.adoc, a Box Plot showing the Distribution of the weight (in kilos) of Young Dogs}
```

Comment:
@schanzer noting that the lesson on box plots doesn't come until after this lesson... should we move this page to the box pots lesson?

Reply:
@flannery-denny I'm not seeing any of this in the lesson plan.

Reply:
fixed!